src/app/users/get-user.ts

```diff
@@ -69,20 +65,20 @@ const updateUserIPsInfo = async ({
         return
       }
 
-      const lastIP = userIP.lastIPs.find((ipObject) => ipObject.ip === ip)
+      const lastIP = userIP.lastIPs?.find((ipObject) => ipObject.ip === ip)
```

Comment:
This change probably not necessary if `lastIPs` is never undefined
```suggestion
      const lastIP = userIP.lastIPs.find((ipObject) => ipObject.ip === ip)
```

Reply:
Fixed 👍